modules/private-cluster/variables.tf

```diff
@@ -116,7 +116,7 @@ variable "ip_range_pods" {
 }
 
 variable "ip_range_services" {
-  description = "The _name_ of the secondary subnet ip range to use for services"
+  description = "The _name_ of the secondary subnet range to use for services"
```

Comment:
Was this stale that needed to be regenerated?

Reply:
This was regenerated for this MR, yes.